docs/source/release/v3.8.0/ui.rst

```diff
@@ -22,6 +22,12 @@ User Interface
 Instrument View
 ###############
 
+Workspace Matrix View
+#####################
+* Masked spectra will be highlighted with background colour (typically light grey).
+* Monitor spectra will have a dynamic background colour depending on the system settings (typically light yellow, as before).
+.. figure::  ../../images/MaskedAndMonitor.png
+
```

Comment:
Yes, please test, it should be grey. To me masking seems more important to highlight, even if it is a monitor.